### DISCUSSION TITLE
Projektimport liefert nur alte Gruppenadressen

### DISCUSSION THREAD

**Discussion by curana** [brutella/hkknx-public#246] [Q&A] [answered]
Hallo,

Ich habe meine aktuelle ETS-Projektdatei importiert, allerdings wird mir eine veraltete Liste importiert. Anscheinend gibt es mehrere Versionen in solch einer Datei? In der ETS ist alles korrekt.

Hat das Problem schon mal jemand gehabt? Kann ich das irgendwie bereinigen?

Danke für Tipps.

**Comment by brutella** [comment]
Wieviele importierte Projekte wird bei dir in hkknx unter Einstellungen angezeigt?

**Comment by curana** [comment]
Ich sehe dort nur eines. Zuvor war auch noch kein Projekt importiert worden, soweit ich mich erinnere.

**Comment by brutella** [comment, marked as answer]
Dann sollten nur die Gruppenadresse angezeigt werden, die im ETS Projekt stehen. Hast du auch einen aktuellen Stand des Projektes von der ETS exportiert?

**Comment by curana** [comment]
Ich habe einen neuen Export gemacht. Mein Fehler.
